sympy/concrete/expr_with_limits.py

```diff
@@ -25,12 +26,12 @@ def _process_limits(*symbols):
     limits = []
     orientation = 1
     for V in symbols:
-        if isinstance(V, Symbol):
+        if isinstance(V, (Symbol, Idx)):
```

Comment:
> I am not very sure. If something like Sum(A[i], i) should work. Looks a little inconsistent in the way Sum usually works.

Well, if _i_ contains information about the range, what's the problem? The way Sum usually works requires to specify a range, because _Symbol_ does not contain that information, while _Idx_ may contain it.